dao/src/integ-test/java/uk/gov/moj/sdt/dao/BulkSubmissionDaoTest.java

```diff
@@ -73,15 +77,18 @@ public class BulkSubmissionDaoTest extends AbstractIntegrationTest {
      */
     private IBulkSubmissionDao bulkSubmissionDao;
 
+    private IBulkCustomerDao bulkCustomerDao;
+    private ITargetApplicationDao targetApplicationDao;
+
     /**
      * Bulk Customer to use for the test.
      */
-    private IBulkCustomer bulkCustomer;
+    private BulkCustomer bulkCustomer;
```

Comment:
Any particular reason the type has been changed from the interface?  Same for targetApplication below.

Reply:
Ya, as we have to use the entity classes here (classes marked with @Entity), so we need to change this to implementation class instead of interface